### DISCUSSION TITLE
Money columns does not exist when using update_all method

### DISCUSSION THREAD

**Discussion by planetaska** [RubyMoney/money-rails#675] [Q&A]
I have a Book model with a `price_cents` column like this:

````
class Book
    monetize :price_cents
end
````

This Book model is a STI base model for other Book types such as Novel, Comic models.

And I can use other money features just fine. But when I need to mass update the books resource, I run into this issue:

Related code in controller:
````
# book_params is a url params submitted by a form
# params content like: {price: 100, title: ""}
books_params = book_params.compact_blank.to_h
@books.update_all(books_params)
````

Gives this error:

````
PG::UndefinedColumn: ERROR: column "price" of relation "books" does not exist LINE 1: UPDATE "books" SET "updated_by" = $1, "price" = $2 WH... ^
````

It seems that Rails couldn't find the correct column name when using update_all.
If I change `update_all` to just `update`, then the code works as expected.

````
@books.each do |book|
    book.update(books_params)
end
````

Obviously this is not ideal. So my question is: is there a way to update Money columns with `update_all` in a STI relation?

**Comment by sunny** [comment]
Using `update_all` does not accept changing Money columns, whether using STI or not. I think this could be a good addition to money-rails.
